src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/DocAssemblyService.java

```diff
@@ -49,6 +51,25 @@ public DocAssemblyService(
         this.judgeTemplateId = judgeTemplateId;
     }
 
+    public CCDDocument generateDocument(String authorisation,
+                                        DocAssemblyTemplateBody formPayload,
+                                        String templateId) {
+
+        DocAssemblyRequest docAssemblyRequest = DocAssemblyRequest.builder()
+            .templateId(templateId)
+            .outputType(OutputType.PDF)
+            .formPayload(formPayload)
+            .build();
+
+        var docAssemblyResponse = docAssemblyClient.generateOrder(
+            authorisation,
+            authTokenGenerator.generate(),
+            docAssemblyRequest
+        );
+
+        return CCDDocument.builder().documentUrl(docAssemblyResponse.getRenditionOutputLocation()).build();
```

Comment:
Would be easier to read if:
```suggestion
        return CCDDocument.builder()
          .documentUrl(docAssemblyResponse.getRenditionOutputLocation())
          .build();
```